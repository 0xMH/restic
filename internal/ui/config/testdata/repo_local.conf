repo "test" {
    backend = "local"
    path = "/foo/bar/baz"
}

foobar "test" {
    x = "y"
}

quiet = false
