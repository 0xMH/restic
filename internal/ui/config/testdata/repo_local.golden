{
  "Repo": "test",
  "Password": "",
  "PasswordFile": "",
  "Backends": {
    "test": {
      "Type": "",
      "Path": "/foo/bar/baz"
    }
  },
  "Backup": {
    "Target": null,
    "Excludes": null
  }
}
